src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAsyncSpillTests.cs

```diff
@@ -6418,5 +6418,41 @@ static async Task Main()
             var verifier = CompileAndVerify(source, expectedOutput: "1");
             verifier.VerifyDiagnostics();
         }
+
+        [Fact, WorkItem(48255, "https://github.com/dotnet/roslyn/issues/48255")]
+        public void MakeRefWithStackalloc()
+        {
+            var source = @"
+using System;
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        var ts = new TestStruct();
+        Console.WriteLine(ts.ToString());
+    }
+  }
+
+unsafe struct TestStruct
+{
+    public override string ToString()
+    {
+        Span<char> chars = stackalloc char[4];
+        var reference = __makeref(this);
+
+        return string.Empty;
+   }
+}
+";
+            var comp = CreateCompilationWithMscorlibAndSpan(source, options: TestOptions.UnsafeDebugExe);
+            comp.VerifyDiagnostics();
+            var verifier = CompileAndVerify(
+                compilation: comp,
+                expectedOutput: string.Empty,
+                verify: Verification.Fails
```

Comment:
I'm not sure what this mean.
I added it because the test failed with "Verification failed" error. Let me know if the test change is incorrect.

Reply:
> I'm not sure what this mean.
> I added it because the test failed with "Verification failed" error. Let me know if the test change is incorrect.

This looks suspicious. It looks like there are existing tests with ```_makeRef``` that do not fail verification. If the scenario uses unverifiable code, it would be better to avoid that.  

---
In reply to: [499164215](https://github.com/dotnet/roslyn/pull/48265#discussion_r499164215) [](ancestors = 499164215)

Reply:
@AlekseyTs I'm not sure how to fix that. ;(